pkg/api/types.go

```diff
@@ -11,18 +11,7 @@ import (
 	"github.com/neondatabase/autoscaling/pkg/util"
 )
 
-// PodName represents the namespaced name of a pod
-//
-// Some analogous types already exist elsewhere (e.g., in the kubernetes API packages), but they
-// don't provide satisfactory JSON marshal/unmarshalling.
-type PodName struct {
-	Name      string `json:"name"`
-	Namespace string `json:"namespace"`
-}
-
-func (n PodName) String() string {
-	return fmt.Sprintf("%s:%s", n.Namespace, n.Name)
-}
+type PodName = util.NamespacedName
```

Comment:
canonically the type `=` (rather than conventional aliasing) exists for maintaining api compatibility during refactors to make it possible to move types from packages without needing to change lots of call sights, and sort of indicates something is still in a state of transition. 

Having said that, I think this is a silly convention, and I think this kind of type declaration can be useful more broadly, but because it's sort of against the convention, I think it bears a comment to explain the intent?

Reply:
> sort of indicates something is still in a state of transition

that's exactly what's going on here :)

`api.PodName` is removed by the second commit.